Microsoft.Toolkit.Uwp.SampleApp/App.xaml

```diff
@@ -3,7 +3,7 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-			 xmlns:not_wasm="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:not_wasm="http://uno.ui/not_wasm"
              mc:Ignorable="d not_wasm"
```

Comment:
not wasm should include windows, so placing it in ignorable is wrong.

Reply:
If you remove it from Ignorable, then you need to keep  `xmlns:not_wasm="http://schemas.microsoft.com/winfx/2006/xaml/presentation"`

Reply:
Alright fixed!